management/dns_update.py

```diff
@@ -228,6 +228,10 @@ def has_rec(qname, rtype, prefix=None):
 	defaults = [
 		(None,  "A",    env["PUBLIC_IP"],       "Required. May have a different value. Sets the IP address that %s resolves to for web hosting and other services besides mail. The A record must be present but its value does not affect mail delivery." % domain),
 		(None,  "AAAA", env.get('PUBLIC_IPV6'), "Optional. Sets the IPv6 address that %s resolves to, e.g. for web hosting. (It is not necessary for receiving mail on this domain.)" % domain),
+		("mta-sts",  "A", env["PUBLIC_IP"], "Required. For MTA-STS verification."),
```

Comment:
The descriptions here are for the External DNS page of the control panel and are tips for people managing their own DNS. These should actually say Optional at the start since they are not required for mail deliverability, and they should not that the record should only be used if the HTTP part of the MTA-STS specification is implemented too.

Reply:
Let me know any further tweaks you would like to the wording here.  See https://github.com/mail-in-a-box/mailinabox/pull/1556/commits/5cb6fcce367b050cd72c71d83b15528ab8d1a283

I added an additional comment on the _smtp._tls TXT record as this is optional optional, it is not necessary for the MTA-STS to function correctly and does request/cause email reports to be sent to the postmaster@ email account by all mail servers that have connected to the box.  These reports might be confusing to some and if you receive email from a large number of servers could result in a large number of additional emails (I have only received these from Google so far as I do not have many contacts).  The way this is currently set up I believe these emails are received daily (as the requested by the MTA-STS policy which currently is set at 86400 seconds).  It is possible to increase this number but I do not have any experience in doing this, maybe somebody else has?